api/v1/serializers/members-serializer.js

```diff
@@ -0,0 +1,89 @@
+const appRoot = require('app-root-path');
+const decamelize = require('decamelize');
+const JsonApiSerializer = require('jsonapi-serializer').Serializer;
+const _ = require('lodash');
+
+const { serializerOptions } = appRoot.require('utils/jsonapi');
+const { openapi } = appRoot.require('utils/load-openapi');
+const { apiBaseUrl, resourcePathLink, paramsLink } = appRoot.require('utils/uri-builder');
+
+const memberResourceProp = openapi.definitions.MemberResource.properties;
+const memberResourceType = memberResourceProp.type.enum[0];
+const memberResourceKeys = _.keys(memberResourceProp.attributes.properties);
+const memberResourcePath = 'members';
+const memberResourceUrl = resourcePathLink(apiBaseUrl, memberResourcePath);
+
+/**
+ * The column name getting from database is usually UPPER_CASE.
+ * This block of code is to make the camelCase keys defined in openapi.yaml be
+ * UPPER_CASE so that the serializer can correctly match the corresponding columns
+ * from the raw data rows.
+ */
+_.forEach(memberResourceKeys, (key, index) => {
+  memberResourceKeys[index] = decamelize(key).toUpperCase();
+});
+
+const memberConverter = (rawMembers) => {
+  _.forEach(rawMembers, (member) => {
+    member.MEMBER_LEVEL = parseInt(member.MEMBER_LEVEL, 10);
+    member.MEMBER_EXP_OVER_LEVEL = parseInt(member.MEMBER_EXP_OVER_LEVEL, 10);
+  });
+};
+
+const singleMemberConverter = (rawMember) => {
+  rawMember.MEMBER_LEVEL = parseInt(rawMember.MEMBER_LEVEL, 10);
+  rawMember.MEMBER_EXP_OVER_LEVEL = parseInt(rawMember.MEMBER_EXP_OVER_LEVEL, 10);
+};
```

Comment:
It looks like the logic for `memberConverter` and `singleMemberConverter` is almost identical. You should be able to combine this into a single function. You might have to change how you call it though